README.md

```diff
@@ -68,13 +68,23 @@ python3 DAGify.py --help
 ---
 
 ### Run DAGify (From Source Container)
+You can run DAGify via the CLI or the UI by setting the RUN_MODE environment variable.
+When you run either of the below commands you will mount your current working directory to the container for execution.
+
+#### With CLI
 First you should update your .env.example file to use the environment variables you need. 
-When you run the below commands you will mount your current working directory to the container for execution.
-#### With Defaults
 ```bash
 
 docker build -t localhost/dagify:source . 
-docker run -it --env-file=.env.example -v $(pwd):/app localhost/dagify:source
+docker run -p 8000:8000 -it --env-file=.env.example -v $(pwd):/app -e RUN_MODE=cli localhost/dagify:source;
+
+```
+
+#### With User Interface
+```bash
+
+docker build -t localhost/dagify:source . 
+docker run -p 8000:8000 -it --env-file=.env.example -v $(pwd):/app -e RUN_MODE=ui localhost/dagify:source
```

Comment:
does the Docker run with UI actually need the env file and volume mount as parameters?

Reply:
I expect the underlying code is still expecting the mount dow writing the files that would then be served up for downloading.

Reply:
just cause of the way it's using the existing code.

Reply:
Yes, the UI does not need env file or volume mount.